src/locales/pt-BR.js

```diff
@@ -0,0 +1,189 @@
+export default {
+  'navbar.lang': 'Português',
```

Comment:
Can you change to 'English' ?
Convenient for people who don't know Português to switch to a familiar language

Reply:
People need to know how to change the language and with "Português" do not understand that there is the place to change? Is that what you mean?

Currently it looks like this:
![captura de tela 2018-09-25 as 07 57 19](https://user-images.githubusercontent.com/1683922/46010645-d58b3700-c099-11e8-8eb3-8549219a5176.png)

What you think about this (from umijs.org):
![captura de tela 2018-09-25 as 08 01 18](https://user-images.githubusercontent.com/1683922/46010752-20a54a00-c09a-11e8-89d2-b6185c814d53.png)

And changing the locale file to `'navbar.lang': 'Languages'`

Reply:
you are right